netket/experimental/sampler/metropolis_pt.py

```diff
@@ -60,19 +60,38 @@ def __init__(
         self.beta_diffusion = jnp.zeros((n_chains,), dtype=float)
         self.exchange_steps = jnp.zeros((), dtype=int)
         super().__init__(σ, rng, rule_state)
+        self.n_accepted_proc = jnp.zeros(
+            n_chains, dtype=int
+        )  # correct shape is (n_chains,) and not (n_batches,)
 
     def __repr__(self):
         if self.n_steps > 0:
-            acc_string = "# accepted = {}/{} ({}%), ".format(
-                self.n_accepted, self.n_steps, self.acceptance * 100
-            )
+            acc_string = f"# accepted = {self.n_accepted}/{self.n_steps} ({self.acceptance * 100}%), "
         else:
             acc_string = ""
 
-        text = "MetropolisNumpySamplerState(" + acc_string + f"rng state={self.rng})"
-        text = "MetropolisPtSamplerState(" + acc_string + f"rng state={self.rng}"
+        text = (
+            f"MetropolisPtSamplerState(# replicas = {self.beta.shape[-1]}, "
+            + acc_string
+            + f"rng state={self.rng}"
+        )
         return text
 
+    @property
+    def normalized_diffusion(self):
+        r"""
+        Average variance of the position of :math:`\\beta = 1`.
+        In the ideal case, this quantity should be of order ~[0.2, 1.0]
+        """
+        return jnp.sqrt(self.beta_diffusion / self.exchange_steps / self.beta.shape[-1])
+
+    @property
+    def normalized_position(self):
+        r"""
+        Average position of :math:`\\beta = 1`, normalized and centered around 0.
+        """
+        return self.beta_position / float(self.beta.shape[-1] - 1) - 0.5
+
```

Comment:
those quantities are vectors (per-chain). Would it make more sense/be more useful to report the average across chains?

Reply:
ah yes indeed, I will correct that